### PR TITLE
PR test 1

### PR DESCRIPTION
**Pull request by m-dati**
test of fork sync

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # Hello-Pillo
 experiments and projects
+
+new line for test 29/11/22
```
Comment (Pupillo) [inline README.md:4]: check ok

**Review by Pupillo — DISMISSED** [review]
change  test

**Comment by Pupillo** [comment]
Checking gh functionalities

File: newfile.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+New data
```
Comment (Pupillo) [inline newfile.txt:1]: try change this line

**Review by Pupillo — DISMISSED** [review]

**Comment by Pupillo** [comment]
testing request changes

File: newfile.txt (hunk 2 above)
Comment (Pupillo) [reply newfile.txt:1]: set Chg.Req.

**Comment by m-dati** [comment]
Last update ok

**Review by m-dati — COMMENTED** [review]
test2

File: newfile.txt (hunk 2 above)
Comment (m-dati) [reply newfile.txt:1]: test comment

**Comment by Pupillo** [comment]
Going to merge.

**Pull request merged**